src/typings/Types.ts

```diff
@@ -38,21 +40,21 @@ export interface ProcessedTargetsContext {
    */
   actionFinalStages: ModifierStage[];
   currentSelections: SelectionWithEditor[];
-  currentEditor: vscode.TextEditor | undefined;
+  currentEditor: TextEditor | undefined;
   hatTokenMap: ReadOnlyHatMap;
   thatMark: Target[];
   sourceMark: Target[];
-  getNodeAtLocation: (location: Location) => SyntaxNode;
+  getNodeAtLocation: (location: vscode.Location) => SyntaxNode;
```

Comment:
Hmm. I guess we revisit this once we have parse tree in monorepo

Tempted to wrap it tho for now to contain VSCode dep. 

Maybe add it to document?

Reply:
I was thinking of adding it to ide for now